idp-service/src/test/resources/amr_acr_mapping.json

```diff
@@ -2,13 +2,15 @@
   "amr" : {
     "PIN" :  [{ "type": "PIN" }],
     "OTP" :  [{ "type": "OTP" }],
-    "Inji" :  [{ "type": "INJI" }],
+    "local-face-auth" :  [{ "type": "LFA" }],
```

Comment:
This is now getting renamed to from LFA to WFA - wallet-face-auth, I think AMR also we can keep the same

Reply:
Done

Reply:
changed